doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -1,84 +1,92 @@
+if more fast storage is available, it makes more sense to provision a DB
+device. Because the BlueStore journal is always placed on the fastest device
+available, using a DB device provides the same benefit that using a WAL device
+would, while *also* allowing additional metadata to be stored off the primary
+device (provided that it fits). DB devices make this possible because whenever
+a DB device is specified but an explicit WAL device is not, the WAL will be
+implicitly colocated with the DB on the faster device.
+
+To provision a single-device (colocated) BlueStore OSD, run the following
+command:
 
 .. prompt:: bash $
 
    ceph-volume lvm prepare --bluestore --data <device>
 
-To specify a WAL device and/or DB device:
+To specify a WAL device or DB device, run the following command:
    
 .. prompt:: bash $
 
    ceph-volume lvm prepare --bluestore --data <device> --block.wal <wal-device> --block.db <db-device>
 
-.. note:: ``--data`` can be a Logical Volume using  *vg/lv* notation. Other
-          devices can be existing logical volumes or GPT partitions.
+.. note:: The option ``--data`` can take as its argument any of the the
+   following devices: logical volumes specified using *vg/lv* notation,
+   existing logical volumes, and GPT partitions.
+
+
 
 Provisioning strategies
 -----------------------
-Although there are multiple ways to deploy a BlueStore OSD (unlike Filestore
-which had just one), there are two common arrangements that should help clarify
-the deployment strategy:
+
+BlueStore differs from Filestore in that there are several ways to deploy a
+BlueStore OSD. However, the overall deployment strategy for BlueStore can be
+clarified by examining just these two common arrangements:
 
 .. _bluestore-single-type-device-config:
 
 **block (data) only**
 ^^^^^^^^^^^^^^^^^^^^^
-If all devices are the same type, for example all rotational drives, and
-there are no fast devices to use for metadata, it makes sense to specify the
-block device only and to not separate ``block.db`` or ``block.wal``. The
-:ref:`ceph-volume-lvm` command for a single ``/dev/sda`` device looks like:
+If all devices are of the same type (for example, they are all rotational
+drives), and if there are no fast devices available for the storage of
+metadata, then it makes sense to specify the block device only and to leave
+``block.db`` and ``block.wal`` unseparated. The :ref:`ceph-volume-lvm` command
+for a single ``/dev/sda`` device is as follows:
 
 .. prompt:: bash $
 
    ceph-volume lvm create --bluestore --data /dev/sda
 
-If logical volumes have already been created for each device, (a single LV
-using 100% of the device), then the :ref:`ceph-volume-lvm` call for an LV named
-``ceph-vg/block-lv`` would look like:
+If each device already has a logical volume that has been created for that
+device and that logical volume is using 100% of the device, then the
+:ref:`ceph-volume-lvm` call for an logical volume named ``ceph-vg/block-lv`` is
+as follows:
```

Comment:
There's been a lot of confusion over this.  I *think* is is about pre-created LVMs, so maybe

>If the devices to be used for a BlueStore OSD are pre-created logical volumes, then the

Reply:
Accepted.